lib/kbsecret/session.rb

```diff
@@ -92,6 +92,27 @@ def delete_record(label)
       records.delete(record)
     end
 
+    # Delete an array of records from the session, if they exist. Does nothing
+    # if no such record can be found.
+    # @param labels [Array<[String, :Symbol]>] the labels of the records to delete
+    # @return [void]
+    def delete_records(labels)
```

Comment:
This feature shouldn't require any new API surface -- all the changes should happen in the CLI.